lib/util/json-log-humanizer.js

```diff
@@ -4,37 +4,32 @@ var _         = require('lodash');
 
 var typeProcessors = {};
 typeProcessors.message = function(data) {
-  return data.message;
+  return data.message + extraData(data);
 };
 
 typeProcessors.request = function(data) {
-  var requestLine = data.method + " " + data.url;
+  var requestLine = data.method + " " + data.url + extraData(data);
   var bodyJson    = JSON.stringify(data.body, null, 2);
   return requestLine + "\n" + bodyJson;
 };
 
 typeProcessors.response = function(data) {
-  var statusText = http.STATUS_CODES[data.status];
+  var statusText = http.STATUS_CODES[data.status] + extraData(data);
 
   return data.status + " " + statusText + " (" + data.duration.toFixed(3) + "ms)";
 };
 
 typeProcessors.error = function(data) {
-  var initialLine = data.error.className + ": " + data.error.message;
+  var initialLine = data.error.className + ": " + data.error.message + extraData(data);
   var stackLines  = _.map(data.error.stack, function(line) {
                       return "  " + line;
                     }).join("\n");
-  var extras = "  {\n";
```

Comment:
Seems like we were trying to use json here. Not sure the new implementation does that. Also is there a reason we're doing the manual string conversion instead of `JSON.stringify`?